boards/microbit/include/board.h

```diff
@@ -21,20 +21,13 @@
 #define BOARD_H
 
 #include "cpu.h"
+#include "board_common.h"
 #include "periph_conf.h"
```

Comment:
we can drop `cpu.h` (included from `board_common.h`) and `periph_conf.h` (not used) from the includes. Probably this is true for all the other board.h files, too?!